windows-server-containers-preview/Readme.md

```diff
@@ -0,0 +1,14 @@
+## Windows Server Container Host Preview (Docker Ready)
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fneilpeterson%2Fazure-quickstart-templates%2Fmaster%2Fwindows-server-containers-preview%2Fazuredeploy.json" target="_blank">
```

Comment:
I think you had this before the recent changes but please change to Azure repo link :)